lerna.json

```diff
@@ -9,5 +7,11 @@
       ]
     }
   },
-  "packages": ["packages/*"]
-  }
+  "packages": [
+    "packages/*"
+  ],
+  "version": [
+    "independent",
+    "3.15.0"
```

Comment:
I don't think this `"3.15.0"` line needs to be here. The docs state that version key is used to specify the current version of the repo, since we're using independent versioning, we should use:

```
"version": [
  "independent"
]
```

Reply:
updated here https://github.com/cerner/terra-core/pull/2535/commits/74606e6605036404696ff870060dc8d4b1229dee.